scripts/release/mule/package/linux/amd64/deb/package.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -ex
+
+echo
+date "+build_release begin PACKAGE DEB stage %Y%m%d_%H%M%S"
+echo
+
+export BUILD_DEB=1
+export NO_BUILD=1
+export REPO_DIR=/projects/go-algorand
+export GOPATH="$REPO_DIR/go"
+export PATH="$GOPATH:/usr/local/go/bin:$PATH"
+# TODO: Don't hardcode this path!
+# Will change to something like:
+# "$REPO_DIR/tmp/node_pkgs/$OS_TYPE/$ARCH/pkg"
+export PKG_ROOT="$REPO_DIR/tmp/node_pkgs/linux/amd64/deb"
+export VARIATIONS=linux/amd64
+"$REPO_DIR/scripts/build_packages.sh" "$VARIATIONS"
```

Comment:
Can we add an option to pass in a PKG_ROOT with already built binaries instead of having this script always build the tarballs? Also it looks like the docker command is commented out

Reply:
So, that *shouldn't* be rebuilding the tarballs again since the NO_BUILD flag is set, but I haven't verified that.  And yes, that could be an option.

I commented out the docker build b/c it should probably be built by another `mule` task.